Formula/pilosa.rb

```diff
@@ -15,13 +17,23 @@ class Pilosa < Formula
   depends_on "go" => :build
   depends_on "glide" => :build
 
+  go_resource "github.com/rakyll/statik" do
```

Comment:
It would be far preferable to use a Go vendor directory for this.

glide, govendor, godep, and dep are some good choices for managing a vendor directory.